Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "signal_mean = 250\n",
+    "signal = stats.poisson.rvs(signal_mean,size=100000)\n",
+    "plt.hist([signal, r], bins = 400, color=['red','blue'],label=['Signal', 'Background'])\n",
+    "plt.title(\"Backgroung and Signal Prior to Injection\")\n",
+    "plt.xlabel(\"Measurement\")\n",
+    "plt.ylabel(\"Counts\")\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the plot above, we can confirm that signal is a Poisson distribution with a mean of 250. We can also see that there is no overlab between the two distributions here. Let's see what happens when we inject the signal into our background."
```

Comment:
Problem 2A explanation below graph:
- typos: overlab --> overlap
- Otherwise makes sense to me. Good pointing out no overlap. I like this+the graph as a lead in to the next section.